2021-11-04 10:41:02,284 data_manager.py[line:1364] INFO {'############################发送抽水数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '', 'capacity': '1000', 'deep': 0.5, 'bottleNum': 0, 'jwd': '[114.524096, 30.506853]', 'gjwd': '[114.524096, 30.506853]'}}
2021-11-04 10:41:43,001 data_manager.py[line:1364] INFO {'############################发送抽水数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '', 'capacity': '1000', 'deep': 0.5, 'bottleNum': 0, 'jwd': '[114.524096, 30.506853]', 'gjwd': '[114.524096, 30.506853]'}}
2021-11-04 10:41:50,250 data_manager.py[line:1364] INFO {'############################发送抽水数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '', 'capacity': '1000', 'deep': 0.5, 'bottleNum': 0, 'jwd': '[114.524096, 30.506853]', 'gjwd': '[114.524096, 30.506853]'}}
2021-11-04 10:43:59,429 data_manager.py[line:1365] INFO {'############################发送抽水数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '', 'capacity': '1000', 'deep': 0.5, 'bottleNum': 0, 'jwd': '[114.524096, 30.506853]', 'gjwd': '[114.524096, 30.506853]'}}
2021-11-04 12:58:01,521 data_manager.py[line:1372] INFO {'############################发送抽水数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '', 'capacity': '1000', 'deep': 0.5, 'bottleNum': 0, 'jwd': '[114.524096, 30.506853]', 'gjwd': '[114.524096, 30.506853]'}}
2021-11-09 18:19:08,250 data_manager.py[line:1451] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.523823, 30.506091]', 'gjwd': '[114.523823, 30.506091]'}}
2021-11-09 18:30:32,989 data_manager.py[line:1452] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.523823, 30.506089]', 'gjwd': '[114.523823, 30.506089]'}}
2021-11-09 18:37:42,540 data_manager.py[line:1453] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.52414, 30.506472]', 'gjwd': '[114.524149, 30.506511]'}}
2021-11-09 18:37:48,771 data_manager.py[line:1453] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.52382, 30.506088]', 'gjwd': '[114.52382, 30.506088]'}}
2021-11-09 18:41:07,201 data_manager.py[line:1454] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.524147, 30.506504]', 'gjwd': '[114.524145, 30.506509]'}}
2021-11-09 18:41:13,414 data_manager.py[line:1454] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.52382, 30.506088]', 'gjwd': '[114.52382, 30.506088]'}}
2021-11-09 18:44:05,528 data_manager.py[line:1454] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.524132, 30.506465]', 'gjwd': '[114.524149, 30.506512]'}}
2021-11-09 18:44:11,760 data_manager.py[line:1454] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.523823, 30.506089]', 'gjwd': '[114.523823, 30.506089]'}}
2021-11-09 19:38:36,791 data_manager.py[line:1455] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.524151, 30.506513]', 'gjwd': '[114.524151, 30.506513]'}}
2021-11-09 19:38:45,229 data_manager.py[line:1455] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.52382, 30.506088]', 'gjwd': '[114.52382, 30.506088]'}}
2021-11-09 19:45:46,773 data_manager.py[line:1450] INFO {'发送检测数据error': ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.199.186', port=8009): Max retries exceeded with url: /union/admin/xxl/data/sampling/save (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022B5C967A08>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))"))}
2021-11-09 19:45:46,787 data_manager.py[line:1451] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.524133, 30.506495]', 'gjwd': '[114.524133, 30.506495]'}}
2021-11-09 19:51:35,123 data_manager.py[line:1454] INFO {'发送检测数据error': ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.199.186', port=8009): Max retries exceeded with url: /union/admin/xxl/data/sampling/save (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021ED7306708>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))"))}
2021-11-09 19:51:35,124 data_manager.py[line:1455] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.524136, 30.506503]', 'gjwd': '[114.524136, 30.506503]'}}
2021-11-09 19:53:24,328 data_manager.py[line:1456] INFO {'发送检测数据error': ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.199.186', port=8009): Max retries exceeded with url: /union/admin/xxl/data/sampling/save (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000017BBD7A1348>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))"))}
2021-11-09 19:53:24,341 data_manager.py[line:1457] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.52415, 30.506513]', 'gjwd': '[114.52415, 30.506513]'}}
2021-11-09 19:53:53,747 data_manager.py[line:1456] INFO {'发送检测数据error': ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.199.186', port=8009): Max retries exceeded with url: /union/admin/xxl/data/sampling/save (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000017BBD798648>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))"))}
2021-11-09 19:53:53,808 data_manager.py[line:1457] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.523821, 30.506092]', 'gjwd': '[114.523821, 30.506092]'}}
2021-11-09 19:59:15,790 data_manager.py[line:1454] INFO {'发送检测数据error': ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.199.186', port=8009): Max retries exceeded with url: /union/admin/xxl/data/sampling/save (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000280FFFA6808>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))"))}
2021-11-09 19:59:15,805 data_manager.py[line:1455] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.523816, 30.50609]', 'gjwd': '[114.523816, 30.50609]'}}
2021-11-09 19:59:41,107 data_manager.py[line:1454] INFO {'发送检测数据error': ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.199.186', port=8009): Max retries exceeded with url: /union/admin/xxl/data/sampling/save (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000280FFFB1108>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))"))}
2021-11-09 19:59:41,109 data_manager.py[line:1455] INFO {'发送检测数据': {'deviceId': 'XXLJC4LCGSCSD1DA002', 'mapId': '1434770242236428290', 'capacity': 1000, 'deep': 0.5, 'bottle_num': 0, 'jwd': '[114.524142, 30.506476]', 'gjwd': '[114.524142, 30.506476]'}}
